### PR TITLE
[aavegotchi] newer staking pool power

### PR DESCRIPTION
**Pull request by candoizo**
Changes proposed in this pull request:
- add logic to calculate new staking pools

### CODE REVIEW

**Comment by ChaituVR** [comment]
Hi @candoizo Existing proposals could be affected by this change, especially the proposal that is open 
https://snapshot.org/#/aavegotchi.eth/proposal/0x396844d504394f7f335f070d443c33455300ee21d90db6c4b089760a3e0469bf

It is better to create a `-v2` strategy for this

**Comment by bonustrack** [comment]
There should be a v2 of the strategy if you don't want past proposal to have different results

**Comment by candoizo** [comment]
In this case after reviewing with team the preferred option is to apply the change on the existing strategy

File: src/strategies/aavegotchi/index.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { multicall } from '../../utils';
 
-export const author = 'vfatouros';
-export const version = '0.1.0';
+export const author = 'candoizo';
+export const version = '0.1.1';
 
 const tokenAbi = [
```
Comment (ChaituVR) [inline src/strategies/aavegotchi/index.ts:6]: Please replace ABI with this one 
```
[
  'function balanceOf(address account) view returns (uint256)',
  'function totalSupply() view returns (uint256)',
  'function stakedInCurrentEpoch(address _account) view returns (tuple(address poolAddress, string poolName, string poolU
rl, uint256 rate, uint256 amount)[] _staked)',
  'function staked(address _account) view returns (uint256 ghst_, uint256 poolTokens_, uint256 ghstUsdcPoolToken_)'      
]
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/aavegotchi/index.ts (hunk 1 above)
Comment (candoizo) [reply src/strategies/aavegotchi/index.ts:6]: fixed!

**Review by bonustrack — APPROVED** [review]

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**